srv/salt/_modules/osd.py

```diff
@@ -1244,6 +1250,9 @@ def prepare(self):
         if self.osd.device:
             cmd = "PYTHONWARNINGS=ignore ceph-disk -v prepare "
 
+            # OSD ID
```

Comment:
Maybe a bit more descriptive comment. The other option would be to extend the docstring and explain what the osd_id is used for.

Reply:
Updating